appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -666,7 +666,10 @@ private YaVersion() {
   // For CLOCK_COMPONENT_VERSION 2:
   // - The pattern parameter was added to the FormatDate and FormatDateTime.
   // - Add Duration Support
-  public static final int CLOCK_COMPONENT_VERSION = 3;
```

Comment:
Somehow the numbering in the comments got off by one here (not your fault). Update the comments to indicate that duration support was added in v3 and update your addition to be `// For CLOCK_COMPONENT_VERSION 4`